perceval/backends/core/mattermost.py

```diff
@@ -253,6 +256,12 @@ def _parse_posts(self, raw_posts):
         for post_id in parsed_posts['order']:
             yield parsed_posts['posts'][post_id]
 
+    def _parse_images(self, images):
+        """Parse images and returns a list of images."""
+
+        list_images = [{**images[i], **{'url': i}}for i in images]
```

Comment:
You missed this part (space in between `}` and `for`.

```suggestion
        list_images = [{**images[i], **{'url': i}} for i in images]
```